dockertest/environment.py

```diff
@@ -34,3 +35,18 @@ def set_selinux_context(pwd, context=None, recursive=True):
         raise OSError("Fail to set selinux context by '%s' (%s):\nSTDOUT:\n%s"
                       "\nSTDERR:\n%s" % (_cmd, cmd.poll(), cmd.stdout.read(),
                                          cmd.stderr.read()))
+
+def get_selinux_context(pwd):
+    """
+    When selinux is enabled, return the context of ``pwd``
+    :param pwd: target file or directory
+    :return: SELinux context as a string or None if not set
+    """
+    try:
+        # may include null-termination in string
+        return getxattr(pwd, 'security.selinux').replace('\000', '')
```

Comment:
Probably a better way to do this.  Afraid some newer version of ``xattr`` will strip-null automatically, so didn't want to blindly discard the last character or assume it will be at the end.